WMFData/Sources/WMFData/Data Controllers/Article Tabs/WMFArticleTabsDataController.swift

```diff
@@ -642,38 +642,43 @@ public protocol WMFArticleTabsDataControlling {
     }
     
     public func updateSurveyDataTabsOverviewSeenCount() {
-        var seenCount: Int = (try? userDefaultsStore?.load(key: WMFUserDefaultsKey.articleTabsOverviewOpenedCount.rawValue)) ?? 0
+        var seenCount: Int = (try? userDefaultsStore?.load(key: WMFUserDefaultsKey.articleTabsOverviewOpenedCountBandC.rawValue)) ?? 0
         
         seenCount += 1
-        try? userDefaultsStore?.save(key: WMFUserDefaultsKey.articleTabsOverviewOpenedCount.rawValue, value: seenCount)
+        try? userDefaultsStore?.save(key: WMFUserDefaultsKey.articleTabsOverviewOpenedCountBandC.rawValue, value: seenCount)
     }
-    
+
     public func updateSurveyDataTappedLongPressFlag() {
         try? userDefaultsStore?.save(key: WMFUserDefaultsKey.articleTabsDidTapOpenInNewTab.rawValue, value: true)
     }
     
     public func shouldShowSurvey() -> Bool {
-        // Make sure it's before July 31, 2025
+        // Make sure it's before January 31, 2026 for B and C
         let now = Date()
         let calendar = Calendar.current
-        let deadlineComponents = DateComponents(year: 2025, month: 7, day: 31)
+        let deadlineComponents = DateComponents(year: 2026, month: 1, day: 31)
         
         guard let deadline = calendar.date(from: deadlineComponents),
               now <= deadline else {
             return false
         }
         
-        let seenCount: Int = (try? userDefaultsStore?.load(key: WMFUserDefaultsKey.articleTabsOverviewOpenedCount.rawValue)) ?? 0
-        let didTapLongPress = (try? userDefaultsStore?.load(key: WMFUserDefaultsKey.articleTabsDidTapOpenInNewTab.rawValue)) ?? false
-        let seenSurvey = (try? userDefaultsStore?.load(key: WMFUserDefaultsKey.articleTabsDidShowSurvey.rawValue)) ?? false
+        var seenCount: Int = 0
+        if moreDynamicTabsGroupBEnabled || moreDynamicTabsGroupCEnabled {
```

Comment:
This could be simplified to `needsMoreDynamicTabs`, it'll also make it easier when you have the actual groups assigned, since it'll check the toggle and the assignemnt